run.py

```diff
@@ -13,6 +13,6 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 if __name__ == '__main__':
-    port = os.getenv('FS_DEV_PORT', 8080)
+    port = os.getenv('FS_DEV_PORT', 8082)
```

Comment:
Dockerfile needs updating too

Reply:
This shouldn't affect docker

Reply:
The expose port should reflect the port the app is listening on https://github.com/ONSdigital/ras-frontstage/blob/master/Dockerfile#L6. You'll also need to update the docker compose files https://github.com/ONSdigital/ras-rm-docker-dev/blob/master/.env

Reply:
@benjefferies I've spoken to Rhi - I am going to take a look at ras-rm-docker-dev repo - see if we can align the ports there with the ports in Rhi's list (from link at top)